package.json

```diff
@@ -158,6 +158,7 @@
     "lint-staged": "^11.2.6",
     "mockdate": "^2.0.5",
     "nock": "^13.3.8",
+    "node-fetch": "^3.3.2",
```

Comment:
**question** (non-blocking): Are we moving `note-fetch` from being a dependency to a dev-dependency here?

Reply:
Yeah that's right. I realised now the commit message is a little misleading

Reply:
No worries, feel free to update it if you want. But it's just a chore commit, so I'm easy either way